src/Combinatorics/Matroids/matroids.jl

```diff
@@ -1019,3 +1019,31 @@ function automorphism_group(M::Matroid)
   resize!(I, nrows(I), length(M))
   return automorphism_group(I; action=:on_cols)
 end
+
+@doc raw"""
+    is_quotient(Q1::Matroid,Q2::Matroid)
```

Comment:
Super nitpick, feel free to ignore
```suggestion
    is_quotient(Q1::Matroid, Q2::Matroid)
```